### PR TITLE
Fix/capacitor support

### PR DESCRIPTION
**Pull request by varghesethomase**
The scope of this task is to do the necessary changes required for getting the code push working from the Appcenter client perspective when bundling a Capacitor app. This includes not wrapping a folder within `CodePush` directory when the type of the project in Appcenter is of type React Native.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "appcenter-cli",
+  "name": "@data4life/appcenter-cli",
```
Comment (radibit) [inline package.json]:
```suggestion
  "name": "@d4l/appcenter-cli",
```

So it's better aligned with the rest of our npm packages https://www.npmjs.com/~d4l-team

File: package.json [hunk 2]
```diff
@@ -153,5 +153,8 @@
     "xcode": "3.0.1",
     "xml2js": "0.4.23",
     "yazl": "2.5.1"
+  },
+  "directories": {
```
Comment (radibit) [inline package.json]: Why is that addition needed?

Comment (varghesethomase) [reply package.json]: I think that came as a part of the npm init i did. I'll remove it.

File: src/commands/codepush/lib/codepush-release-command-base.ts [hunk 3]
```diff
@@ -103,24 +103,24 @@ export default class CodePushReleaseCommandBase extends AppCommand {
...
 
     if (this.privateKeyPath) {
-      const appInfo = (
-        await out.progress(
-          "Getting app info...",
-          clientRequest<models.AppResponse>((cb) => client.appsOperations.get(this.app.ownerName, this.app.appName, cb))
-        )
-      ).result;
-      const platform = appInfo.platform.toLowerCase();
+      // const appInfo = (
```
Comment (radibit) [inline src/commands/codepush/lib/codepush-release-command-base.ts:111]: Can you please add a reference in the readme about the exact location of the commented lines and brief explanation of the reason why we have to do that?

**Review by radibit — COMMENTED** [review]
Looks good! Would be also nice to update the README (one sentence at the top) to indicate that this was a forked project and the use case we have with Capacitor.

**Comment by radibit** [comment]
Regarding the npm publishing part, please follow the guide we have https://gesundheitscloud.atlassian.net/wiki/spaces/ENG/pages/1953497105/D4L+NPM+registry+process+workflow I'll provide you with the publish access token.

**Review by radibit — APPROVED** [review]
